### PR TITLE
[doc] add glog curl command 

### PR DESCRIPTION
**Pull request by whitewum**
change v & minloglevel at runtime by http command curl.

It is quit useful for on-site debuging

@Amber1990Zhang help to change to Eng and add to readme.md TOC.

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+也可以通过如下命令将日志级别更改为**最详细**
+
+```bash
+> curl "http://127.0.0.1:12000/set_flags?flag=v&value=4"
+> curl "http://127.0.0.1:12000/set_flags?flag=minloglevel&value=0"
+```
+
+或者**关闭** 所有的日志打印(仅保留 FATAL)
```
Comment (Amber1990Zhang) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: remove extra space

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# 日志
+
+Nebula 使用 [glog](https://github.com/google/glog) 来打印日志，使用 gflag 控制日志的级别，并提供 HTTP 接口在运行时动态改变日志级别，以方便追踪问题。
+
+## 参数说明
+
+### glog 中的两个主要参数
+
+- minloglevel 0-4： 对应的日志级别分别为 INFO(DEBUG)，WARING，ERROR，FATAL。通常在调试环境设置为 0，生产环境设置为1。
```
Comment (Amber1990Zhang) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: space missing before 1

**Comment by whitewum** [comment]
why the toc is updated, but this pr is not merged?
I think we should mention some pr depends on other pr

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# 日志
+
+Nebula 使用 [glog](https://github.com/google/glog) 来打印日志，使用 gflag 控制日志的级别，并提供 HTTP 接口在运行时动态改变日志级别，以方便追踪问题。
+
+## 参数说明
+
+### glog 中的两个主要参数
+
+- minloglevel 0-4： 对应的日志级别分别为 INFO(DEBUG)，WARING，ERROR，FATAL。通常在调试环境设置为 0，生产环境设置为 1。
```
Comment (jude-zhu) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: minloglevel 0-3?  numbers of severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively

refer to  https://github.com/google/glog/blob/master/src/glog/log_severity.h

Comment (Amber1990Zhang) [reply docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: @critical27 please help check

Comment (critical27) [reply docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: It seems the minloglevel 0-4 is DEBUG, INFO, WARNING, ERROR, FALAL. And the `v` is valid when `minloglevel` is 0 (It used for debug by printing more detailed log).

Comment (critical27) [reply docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: BTW, the doc here 0-4 list `INFO(DEBUG)，WARING，ERROR，FATAL` seems a little weird. WARNING is alos a typo.

Comment (Amber1990Zhang) [reply docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: Thanks for your reply, please help update this. @whitewum

**Comment by jude-zhu** [comment]
Jenkins go

**Pull request merged**